documentation/writing-policies.md

```diff
@@ -131,6 +131,24 @@ Operators supported:
 - Equal
 - NotEqual
 
+# Background processing
+Kyverno applies policies in foreground and background mode.
+- `foreground`: leverages admission control webhooks to intercept the resource api-request and apply policy on it.
+- `background`: policy-controller applies policies on the existing resoruces after configured re-conciliation time.
+
+A policy is always enable for `foreground` processing, but `background` processing is configurable using a boolean flag at `{spec.background}`.
```

Comment:
update to  'A policy is always enabled...` 

Does the setting apply to the policy or an individual rule? If its the later, we can say...

"'A policy rule is always enabled...`

Reply:
the setting applies to the policy